src/Serilog.HttpClient/LoggingDelegatingHandler.cs

```diff
@@ -32,16 +30,12 @@ public class LoggingDelegatingHandler : DelegatingHandler
         private readonly RequestLoggingOptions _options;
         private readonly ILogger _logger;
 
-        public LoggingDelegatingHandler(
-            RequestLoggingOptions options,
-            HttpMessageHandler httpMessageHandler = default)
+        public LoggingDelegatingHandler(RequestLoggingOptions options)
         {
             _options = options ?? throw new ArgumentNullException(nameof(options));
-            _logger =  options.Logger?.ForContext<LoggingDelegatingHandler>() ?? Serilog.Log.Logger.ForContext<LoggingDelegatingHandler>();
-
-            InnerHandler = httpMessageHandler ?? new HttpClientHandler();
```

Comment:
This was a recent addition, but it breaks for me with this error:

`System.InvalidOperationException: The 'InnerHandler' property must be null. 'DelegatingHandler' instances provided to 'HttpMessageHandlerBuilder' must not be reused or cached.`

I'm not sure how to best address the concerns in #1

Reply:
I had fixed this bug in silimar manner. I used HttpClientHandler for .net 3 or below and SocketsHttpHandler for .net 3.1 or above for better performance.